projects/eslint-plugin/rules/aui/lib/rules/no-url.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * SPDX-FileCopyrightText: © 2017 Liferay, Inc. <https://liferay.com>
+ * SPDX-License-Identifier: MIT
+ */
+
+const message = 'use the vanilla URL class instead of A.Url';
+
+module.exports = {
+	create(context) {
+		return {
+			NewExpression(node) {
+				if (
+					node.callee.object &&
```

Comment:
I think we would also need to make sure `node.callee` exists. Unless it always exists in NewExpression nodes?

Also, I think we would also need to check for just `A.Url`, without the `new`. I can imagine someone might accidentally forget the `new` and so we should warn that as well.